air/src/constraints/chiplets/mod.rs

```diff
@@ -225,3 +249,27 @@ impl<E: FieldElement> ChipletsFrameExt<E> for &EvaluationFrame<E> {
         self.memory_flag()
     }
 }
+
+// HELPERS
+// ================================================================================================
+
+/// Reduces kernel procedures digests using auxiliary randomness.
+fn reduce_kernel_digests<E>(kernel_digests: &[Word], aux_rand_elements: &AuxRandElements<E>) -> E
+where
+    E: FieldElement<BaseField = Felt>,
+{
+    let alphas = aux_rand_elements.rand_elements();
+    kernel_digests.iter().fold(E::ONE, |acc, digest: &Word| {
+        let digest = digest.to_vec();
+        let cur = alphas[0]
+            + KERNEL_PROC_INIT_LABEL.into()
+            + alphas
+                .iter()
+                .skip(2)
```

Comment:
Why `skip(2)` rather than `skip(1)` (i.e. skipping only `alphas[0]`)?

Reply:
Basically, the way we know compute the reduction per "message" is as follows:

1. We use powers of $\beta$ to reduce each message, i.e., $\mathsf{m} = \sum_i \beta^i m_i$
2. The random challenge for the multi-set check is $\alpha$, i.e., we multiply by $(\alpha + \mathsf{m})$

From the above, and noting that we start the powers of $\beta$ from $i=0$, the above expression follows.